src/gui/optionsdialog.ui

```diff
@@ -3155,19 +3155,22 @@ Disable encryption: Only connect to peers without protocol encryption</string>
                   </widget>
                  </item>
                  <item row="1" column="1">
-                  <widget class="QSpinBox" name="spinRSSMaxArticlesPerFeed">
+                  <widget class="QSpinBox" name="spinRSSFetchDelay">
+                   <property name="suffix">
+                    <string> sec</string>
+                   </property>
                    <property name="maximum">
-                    <number>2147483646</number>
+                    <number>999999</number>
```

Comment:
If possible I would allow the maximum value to avoid potential user complaints about some artificial limit:
```suggestion
                    <number>2147483646</number>
```

Reply:
It won't hurt anyone having the freedom to set high values. Looks like good practice to think of that case
Fixed with https://github.com/qbittorrent/qBittorrent/pull/19801/commits/6fa7c4ea81673318f80929f5fedc5bb53150de97